src/components/grids/TabbedDisplay/index.tsx

```diff
@@ -55,16 +55,12 @@ export type TabbedDisplayProps = {
   tabs: Array<{
     // The name to display in the tab. Also used as the key.
     displayName: string;
-    onSelect?: () => void;
     content?: React.ReactNode;
   }>;
+  onClick: (tabDisplayName: string) => void;
```

Comment:
Maybe want to use a name like `onTabSelected`, to reflect that a tab can be selected by means other than a literal click?

Reply:
Yes, good catch!